analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConditionEvaluatesToConstant.CSharp8.cs

```diff
@@ -258,4 +258,36 @@ public void Init<T>(ref T field)
             }
         }
     }
+
+    public class Tuples
+    {
+        public void DoSomething(bool arg)
+        {
+            var trueValue = true;
+            var tuple = (arg, trueValue);
+
+            if (tuple.arg)
+            {
+            }
+
+            if (tuple.trueValue)    // FN
```

Comment:
nitpick - is there a reason to have so many spaces before the comment here and at line 288?

Reply:
VS tab stops are 4 spaces wide and the `)` ends on one, so next tab produces 4 spaces in this case.

I moved the line 288, as there were two tabs.